.travis/deploy_sbt.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o verbose
+
+if [[ "$PROJECT" == "common" ]]
+then
+  echo "Common lib doesn't have a Docker container; skipping deploy..."
+  exit 0
+fi
+
+export VERSION="${VERSION:-0.0.1}"
+export BUILD_ENV="${BUILD_ENV:-dev}"
```

Comment:
This defaults it to dev so we can’t deploy new prod images by accident, but as a result you’ll need to add an `export BUILD_ENV=prod` to the calling script.